### DISCUSSION TITLE
Py-pde Status Indicator

### DISCUSSION THREAD

**Discussion by retter-berkeley** [zwicker-group/py-pde#611] [Q&A] [answered]
I am trying to use py-pde for a reinforcement learning project, which requires 100s of calls to eq.solve().  However, eq.solve() outputs a status bar when I use it as below:

![status](https://github.com/user-attachments/assets/8d273f2e-7413-4c94-b099-1f8e0b55192a)

Is there a way to turn off or suppress this status bar, or the "loading widget" update?  The status bar is eating memory and slowing down the algorithm.  I have tried many standard ways to suppress output in python, including import shutup/shutup.please() and redirecting output, without success.

thanks!

**Comment by david-zwicker** [comment, marked as answer]
The status bar is called a progress bar in the terminology used by the package. The progress bar is implemented as a [tracker](https://py-pde.readthedocs.io/en/latest/packages/pde.trackers.html), and is enabled by default. To disable the progress bar, simply add the argument `tracker=None` when calling the [`solve` method of the PDE](https://py-pde.readthedocs.io/en/latest/packages/pde.pdes.base.html#pde.pdes.base.PDEBase.solve).

**Comment by retter-berkeley** [comment]
Worked, thanks!
